phase4-profile-entsog/.gitignore

```diff
@@ -0,0 +1,31 @@
+HELP.md
```

Comment:
I doubt that that file is needed. I try to have a single ".gitignore" file in the root folder. Can you please try deleting this file and see if still everything is ignored as before? If not just add it to the root file. Thx